Build fail due to wrong import in domWrapper.ts

There is seems to be an error that causes our build to fail after library update to v2.0.0-rc.14
https://github.com/vuejs/vue-test-utils-next/blob/df6c234d2511f500be996620118c99625a86e165/src/domWrapper.ts#L8

I guess proper line should be:
```typescript
import { VueWrapper } from '.';
```
As there is no `src` fodler in npm package:
![image](https://user-images.githubusercontent.com/118516/132848036-43db0833-d8dd-4666-9467-3e9669ecf5cf.png)

This is broken again in 2.0.0-rc.19, because #1180 added more broken imports, but #1184 was submitted before #1180 was merged.

```
../../node_modules/@vue/test-utils/dist/interfaces/wrapperLike.d.ts:1:42 - error TS2307: Cannot find module 'src/constants/dom-events' or its corresponding type declarations.

1 import { DomEventNameWithModifier } from 'src/constants/dom-events';
                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~

../../node_modules/@vue/test-utils/dist/interfaces/wrapperLike.d.ts:2:32 - error TS2307: Cannot find module 'src/createDomEvent' or its corresponding type declarations.

2 import { TriggerOptions } from 'src/createDomEvent';

```

--- a/src/interfaces/wrapperLike.ts
+++ b/src/interfaces/wrapperLike.ts
@@ -1,5 +1,5 @@
-import { DomEventNameWithModifier } from 'src/constants/dom-events'
-import { TriggerOptions } from 'src/createDomEvent'
+import { DomEventNameWithModifier } from '../constants/dom-events'
+import { TriggerOptions } from '../createDomEvent'
 import {
   DefinedComponent,
   FindAllComponentsSelector,

